apps/aechannel/src/aesc_fsm.erl

```diff
@@ -14,6 +14,7 @@
          upd_call_contract/2,     %%
          get_contract_call/4,     %% (fsm(), contract_id(), caller(), round())
          leave/1,
+         close_solo/1,
```

Comment:
I understand that this is not exposed to the WebSocket endpoint yet, hence:
* It has no doc update in https://github.com/aeternity/protocol/blob/362d566af481214a5f650385b10d6cb3cfd0e831/node/api/channel_ws_api_json-rpc.md yet
* It has no line in release notes

Reply:
This is correct